public/javascripts/quiz.js

```diff
@@ -3,6 +3,13 @@ var starters,bench, roster,
     stop_counter = false;
     team_container = $('.team-container'),
     answer_container = $('.answer-container');
+if (typeof(roster) != undefined){
```

Comment:
you cant check length of an undefined variable. You would check length of something that will never be defined. Like a jquery dom pull or something